pkg_building.Rmd

```diff
@@ -57,7 +57,7 @@ The [`tabulizer` package](https://github.com/ropensci/tabulizer) e.g. has an int
 
 ## Code Style
 
-* For more information on how to style your code, name functions, and R scripts inside the `R/` folder, we recommend reading the [code chapter in Hadley's book](http://r-pkgs.had.co.nz/r.html). We recommend the [`styler` package](https://github.com/r-lib/styler) for automating part of the code styling.
+* For more information on how to style your code, name functions, and R scripts inside the `R/` folder, we recommend reading the [code chapter in Hadley's book](http://r-pkgs.had.co.nz/r.html). We recommend the [`styler` package](https://github.com/r-lib/styler) for automating part of the code styling. You can choose to use `=` over `<-` as long you are consistent with one choice within your package. 
```

Comment:
Maybe `or` instead of `over` in `= over <-` just to be clear we don't have a preference? or do we want to signal that `<-` is slightly better?

Reply:
don't we have a preference?

Reply:
i guess we do yeah